utt/utt-client-api/src/User.cpp

```diff
@@ -447,9 +447,13 @@ void User::updateTransferTx(const Transaction& tx, const TxOutputSigs& sigs) {
 
     // Claim coins
     auto claimedCoins = pImpl_->client_->claimCoins(uttTx, pImpl_->params_, sigs);
-
+    bool invalidCoinsInTransfer(false);
```

Comment:
use the regular convention here 
either `invalidCoinsInTransfer = false` or `invalidCoinsInTransfer{false}`

Reply:
I'll update in the next commit.